plugin/thematic.vim

```diff
@@ -88,6 +88,21 @@ else
   let g:thematic#theme_name = ''
 endif
 
+if !exists('g:Lf_Extensions')
+  let g:Lf_Extensions = {}
+endif
+
+let g:Lf_Extensions.thematic = {
```

Comment:
Why is this stuff getting defined as a global? Why do people without LeaderF need this in their runtime variable namespace?

Reply:
give me some time... I also think perhaps use exists('g:leader_loaded') to determine whether define this variable can be taken into consideration.

Reply:
how do you think 
```{.vim}
if exists('g:leaderf_loaded')
......
endif
```
?

for any user who don't use leaderf. they will not have 'g:leaderf_loaded'.